src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,89 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesQuantity = Math.ceil(total / perPage);
+  const arrayOfPages = getNumbers(1, pagesQuantity);
+
+  return (
+    <>
+      <ul
+        className="pagination"
+      >
+        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions */}
```

Comment:
add onClick to `a` tag to avoid errors (always try to figure out, google lint error before disabling it)

Reply:
done